### PR TITLE
add some specifications on the top command

### PR DESCRIPTION
**Pull request by tianshapjq**
**What this PR does / why we need it**:
the top command is available when heapster works fine on the server, but the specifications doesn't mention about this. So I added some specifications here.

### CODE REVIEW

File: pkg/kubectl/cmd/top.go [hunk 1]
```diff
@@ -32,13 +32,15 @@ var (
 	topLong = templates.LongDesc(`
 		Display Resource (CPU/Memory/Storage) usage.
 
-		The top command allows you to see the resource consumption for nodes or pods.`)
+		The top command allows you to see the resource consumption for nodes or pods.
+
+		This command is available only if the Heapster works fine on the server. `)
```
Comment (fabianofranz) [inline pkg/kubectl/cmd/top.go]: "This command requires Heapster to be correctly configured and working on the server."

File: pkg/kubectl/cmd/top.go [hunk 2]
```diff
@@ -32,13 +32,15 @@ var (
...
 )
 
 func NewCmdTop(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "top",
-		Short: "Display Resource (CPU/Memory/Storage) usage",
+		Short: "Display Resource (CPU/Memory/Storage) usage, available only if Heapster works fine on the server.",
```
Comment (fabianofranz) [inline pkg/kubectl/cmd/top.go]: No need to mention it in the short description, long desc should be enough.

**Review by fabianofranz — CHANGES_REQUESTED** [review]

File: pkg/kubectl/cmd/top.go (hunk 2 above)
Comment (tianshapjq) [inline pkg/kubectl/cmd/top.go]: ok, done :)

**Comment by fabianofranz** [comment]
@k8s-bot ok to test

**Review by fabianofranz — APPROVED** [review]

**Comment by fabianofranz** [comment]
@tianshapjq looks good, thanks!

**Pull request merged**